packages/openapi-typescript/CONTRIBUTING.md

```diff
@@ -24,14 +26,14 @@ However, if adding a feature or breaking change, please **open an issue first to
 
 ### Writing code
 
-Create a new branch for your PR with `git checkout -b your-branch-name`. Add the relevant code as well as docs and tests. When you push everything up (`git push`), navigate back to your repo GitHub and you should see a prompt to open a new PR.
+Create a new branch for your PR with `git checkout -b your-branch-name`. Add the relevant code as well as docs and tests. When you push everything up (`git push`), navigate back to your repo in GitHub and you should see a prompt to open a new PR.
 
 While best practices for commit messages are encouraged (e.g. start with an imperative verb, keep it short, use the body if needed), this repo doesn’t follow any specific guidelines. Clarity is favored over strict rules. Changelogs are generated separately from git (see [the Changelogs section](#changelogs))
 
 When working locally, run:
 
 ```bash
-npm run dev
+pnpm run dev
```

Comment:
In 95% of cases this doesn’t make a difference, but every now and then some `npm *` commands will fail in a monorepo. Better to be safe.